### PR TITLE
FIX CRITICAL BUG COSTING JUMPMIND MILLIONS

### PR DESCRIPTION
**Pull request by blubaugha**
### Summary
This fixes a

# mIsSiOn
# cRiTiCaL
# bUg

where the `DialogHeaderComponent` was missing some padding between the text and image.

### Screenshots
**Before**
Screen Shot 2021-08-05 at 5 04 34 PM

# AFTER!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
Screen Shot 2021-08-05 at 5 05 18 PM

# OH
# YEAH!

# NOW THE MONEY COMES ROLLING IN!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

### CODE REVIEW

File: openpos-client-libs/projects/openpos-client-core-lib/src/lib/shared/screen-parts/dialog-header/dialog-header.component.scss [hunk 1]
```diff
@@ -6,6 +6,8 @@
     grid-template-columns: auto auto;
     padding:8px;
     border-radius: 4px;
+    margin-right: 8px;
```
Comment (jchirico8) [inline openpos-client-libs/projects/openpos-client-core-lib/src/lib/shared/screen-parts/dialog-header/dialog-header.component.scss:9]: Am I mistaken or did I just see you remove 8 pixels of margin on a recent PR?

Comment (blubaugha) [reply openpos-client-libs/projects/openpos-client-core-lib/src/lib/shared/screen-parts/dialog-header/dialog-header.component.scss:9]: DUDE, 

#### you
### are
## NOT
# mIsTaKeN
# !

Comment (blubaugha) [reply openpos-client-libs/projects/openpos-client-core-lib/src/lib/shared/screen-parts/dialog-header/dialog-header.component.scss:9]: # BUT IT WAS A DIFFERENT 8px CULPRIT!

Comment (jchirico8) [reply openpos-client-libs/projects/openpos-client-core-lib/src/lib/shared/screen-parts/dialog-header/dialog-header.component.scss:9]: I'm skeptical, but I'll allow it

**Review by jchirico8 — APPROVED** [review]

File: openpos-client-libs/projects/openpos-client-core-lib/src/lib/shared/screen-parts/dialog-header/dialog-header.component.scss (hunk 1 above)
Comment (blubaugha) [reply openpos-client-libs/projects/openpos-client-core-lib/src/lib/shared/screen-parts/dialog-header/dialog-header.component.scss:9]: # Your
# approval
# will

# EARN YOU MILLIONS!!!!!!!!!!!!!!
### (minus tax)

**Comment by blubaugha** [comment]
> @jchirico8 
> I'm skeptical, but I'll allow it

# And Joe's MILLIONS comes ROLLING IN!!!!!!!!!!!!!!!!!

**Pull request merged**